docs/CONFIG.md

```diff
@@ -10030,6 +10030,119 @@ GasLimitDefault = 400000
 
 </p></details>
 
+<details><summary>Hemi Mainnet (43111)</summary><p>
```

Comment:
nit: wonder why this config change is showing up here. Do we maybe need to merge develop and regenerate the config doc?

Reply:
I'm not sure either? I pulled the latest changes but there's no change to the config doc.